dexterity/membrane/behavior/password.py

```diff
@@ -22,6 +22,23 @@
 from zope.interface import provider
 
 
+class BCRYPTEncryptionScheme(object):
+    """A BCRYPT AuthEncoding."""
+
+    def encrypt(self, pw):
+        return bcrypt.hashpw(pw, bcrypt.gensalt())
+
+    def validate(self, reference, attempt):
+        try:
+            valid = bcrypt.hashpw(attempt, reference) == reference
+        except ValueError:
+            valid = False
+        return valid
+
+
+AuthEncoding.registerScheme('BCRYPT', BCRYPTEncryptionScheme())
```

Comment:
I would suggest to remove the pep8 cleanup from that AccessControl pull request. It is now not clear at all that you are sneaking in bcrypt as extra dependency.